test/unit/tasks/publish_missed_scheduled_editions_test.rb

```diff
@@ -0,0 +1,27 @@
+require "test_helper"
+require "rake"
+
+class EditionsPublishMissedScheduledEditionsTaskTest < ActiveSupport::TestCase
+  def setup
+    Rake::Task["editions:publish_missed_scheduled_editions"].reenable
+  end
+
+  test "publishes editions that have missed their scheduled publish time" do
+    missed_edition = FactoryBot.create(:edition, state: "scheduled_for_publishing", publish_at: 1.hour.from_now)
+    missed_edition.update_attribute(:publish_at, 1.hour.ago) # rubocop:disable Rails/SkipsModelValidations
+
+    Rake::Task["editions:publish_missed_scheduled_editions"].invoke
+    missed_edition.reload
+
+    assert_equal "published", missed_edition.state
+  end
+
+  test "do not publish editions with publishing scheduled in the future" do
```

Comment:
Consider renaming test to "does not…" in order to use the same tense as the other test.